src/scales/scale.logarithmic.js

```diff
@@ -119,13 +119,14 @@ module.exports = function(Chart) {
 
 						helpers.each(dataset.data, function(rawValue, index) {
 							var values = valuesPerStack[key];
-							var value = +me.getRightValue(rawValue);
+							var value = me.parseValue(rawValue);
+
 							// invalid, hidden and negative values are ignored
-							if (isNaN(value) || meta.data[index].hidden || value < 0) {
+							if (isNaN(value.val) || meta.data[index].hidden || value.val < 0) {
```

Comment:
validate `max` instead of `val` since it's `max` that's being used

Reply:
done